### PR TITLE
[DEV APPROVED]7804 categories setup

### PR DESCRIPTION
**Pull request by jriga**
**Context**
Global Nav works changes the current categories. This PR does:
- [x] add support for old categories structure
- [x] creates the new categories and assigns them to clump

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
   config.assets.debug = true
+  config.action_controller.asset_host = ENV.fetch('MAS_CMS_PUBLIC_URL', 'http://localhost')
```
Comment (tomas-stefano) [inline config/environments/development.rb]: Is this line intentional?

Comment (tomas-stefano) [inline config/environments/development.rb]: Ok. We already have on the .env file an env var called MAS_CMS_URL. Is this were you're looking for?

**Comment by rnalexander** [comment]
@tomas-stefano If all the changes you reviewed are ok, can you indicate that here?

**Comment by tomas-stefano** [comment]
There is no mas cms public url env var on CMS. So I think that could be WIP Pr.

**Comment by rnalexander** [comment]
Then can you change the title to WIP so it stops cluttering the Pull announcements?

File: config/environments/development.rb (hunk 1 above)
Comment (jriga) [inline config/environments/development.rb]: ♻️

**Comment by jriga** [comment]
the PR is ready now. I've changed the env var after my discussion with @tomas-stefano

**Comment by tomas-stefano** [comment]
👍

**Comment by rnalexander** [comment]
:+1: :shipit:

**Pull request merged**